presets/mainnet/deneb.yaml

```diff
@@ -4,5 +4,5 @@
 # ---------------------------------------------------------------
 # `uint64(4096)`
 FIELD_ELEMENTS_PER_BLOB: 4096
-# `uint64(2**2)` (= 4)
-MAX_BLOBS_PER_BLOCK: 4
+# `uint64(2**4)` (= 16)
```

Comment:
Given how sensitive this is should a note be made somewhere that capacity is actually limit in exec layer?

Reply:
added the note in p2p interface